app/controllers/mentee_profiles_controller.rb

```diff
@@ -1,6 +1,24 @@
 class MenteeProfilesController < ApplicationController
-	def show
-	end
-	def dashboard
-	end
+  def show
+    @user = User.find_by(id: params[:id])
+    @mentor = User.find_by(id: @user.matched_id)
+
+  end
+  def dashboard
+    @mentor = User.find_by(id: current_user.matched_id)
+  end
+  def missing_mentor
+    @user = current_user
+    if !@user.blank? 
```

Comment:
Also added, if mentor is not logged in, and prersses button, then i eneded this case, to ask her to login first.